test/package.json

```diff
@@ -3,7 +3,10 @@
   "version": "0.0.1-0",
   "private": true,
   "scripts": {
-    "test": "node --preserve-symlinks test.js test"
+    "test": "NODE_PATH=./node_modules node --preserve-symlinks test.js test",
```

Comment:
Use NODE_PATH env var to tell node to use the `node_modules` folder in the test folder.

At first, this may seem terrible.  However, it *may* solve an separate issue we've been having.  This needs to be tested.  That issue is that...

In order to get our linting stuff working for devs, we have to do an `npm install` on the root of the module, which installs all of the dependencies.  This is partially due to the fact that doing `npm install --only dev` does not work with eslint.  See: https://github.com/digitalbazaar/eslint-config-digitalbazaar/issues/33

So, having dependencies in the `node_modules` in the root of the project can lead to confusion when devs need to symlink other deps for testing in the test framework. For instance, if a dev creates a symlink for `test/node_modules/edv-client`, `node_modules/edv-client` will still be used instead.